packages/coin-avaxc/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "@coolwallet/avaxc",
+  "version": "1.0.0",
+  "description": "Coolwallet AVAXC sdk",
+  "main": "lib/index.js",
+  "author": "coolwallet-team",
+  "license": "Apache-2.0",
+  "scripts": {
+    "build:types": "tsc --emitDeclarationOnly",
+    "build:ts": "babel src --out-dir lib --extensions \".ts,.tsx\" --source-maps inline",
+    "build": "npm run build:types && npm run build:ts",
+    "test": "jest --verbose"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.7.0",
+    "@babel/core": "^7.7.0",
+    "@babel/plugin-proposal-class-properties": "^7.7.4",
+    "@babel/preset-env": "^7.7.1",
+    "@babel/preset-typescript": "^7.7.7",
+    "@coolwallet/core": "file:../core",
```

```suggestion
    "@coolwallet/core": "^1.1.5",
```

Change the location to npm registry.